virtualizor.py

```diff
@@ -351,15 +359,18 @@ class Host(object):
 """
 
     def __init__(self, hypervisor, conf, definition,
-                 install_server_info, gateway_info):
+                 install_server_info, gateway_info, emulator):
```

Comment:
emulator should be an attribue of hypervisor that we initialize in the contructor.